runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -1419,7 +1434,7 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          J9Method *j9method = NULL;
             {
             TR::VMAccessCriticalSection getResolvedHandleMethod(fe);
-            j9method = jitGetImproperInterfaceMethodFromCP(fe->vmThread(), mirror->cp(), cpIndex);
+            j9method = jitGetImproperInterfaceMethodFromCP(fe->vmThread(), mirror->cp(), cpIndex, NULL);
```

Comment:
I would like some re-assurance that NULL is the proper solution here. PR #6993 changed the way resolved methods are created I would like to make sure that this is reflected in the jitaas solution